tools/clang/test/HLSLFileCheck/hlsl/types/cast/StructCast-strictudt.hlsl

```diff
@@ -0,0 +1,48 @@
+// RUN: %dxc -E main -T vs_6_0 -strict-udt-casting %s | FileCheck %s
+
+// CHECK: SV_RenderTargetArrayIndex or SV_ViewportArrayIndex from any shader feeding rasterizer
+
+struct Vertex
+{
+    float4 position     : POSITION0;
+    float4 color        : COLOR0;
+
+uint   RTIndex      : SV_RenderTargetArrayIndex;
+};
+
+struct Interpolants
+{
+    float4 position     : SV_POSITION0;
+    float4 color        : COLOR0;
+
+uint   RTIndex      : SV_RenderTargetArrayIndex;
```

Comment:
nit: weird indentation

Reply:
Another copy of an existing test that has exactly the same contents, besides the extra flag and necessary explicit casting.  I could fix this, but should I fix the original as well?  I kind of don't want to crack it open right now though.